outputs.tf

```diff
@@ -28,26 +28,6 @@ output "elasticache_cluster_id" {
   description = "Id of the ElastiCache cluster"
 }
 
-output "rds_endpoint_address" {
```

Comment:
Secrets manager is a good move, though removing these outputs will break the tests.

Reply:
Hello, thank you so much for open sourcing this; it's a truly wonderful project! 

I am planning on using a modified version of this for something coming up that'll be a similar size in terms of participants. I decided to do a PR before making the modifications mentioned, and do plan to fix the tests so that they no longer break - it's a matter of finding the time to fix those and the fact that I haven't worked with `terratest` before.

In terms of going the EKS route, I think would be a really great idea. Happy to work with you on that in the future if you  are looking for contributors - I've terraformed an EKS deployment that I plan to use to replace my current blog, which I built using https://registry.terraform.io/modules/terraform-aws-modules/eks/aws/latest. 

Lastly, I have some github actions work I was thinking about adding to this PR (or a different one) after checking with you. Basically will run `docs`, `fmt`, `tflint`, and `checkov` as part of the commit process (uses a lot of features from https://github.com/antonbabenko/pre-commit-terraform). Only issue is the last run will require some fixes or checkov statements to address some of the findings.

Reply:
If you are interested I've pushed my initial changes for the EKS backed version to this branch https://github.com/1nval1dctf/terraform-aws-ctfd/tree/feature/eks_and_modules.

I'll have a look at the github actions when i get some more spare time, they seem like a good idea.

Reply:
Apologies, been real busy. Would love to check out what you've got so far, but the branch doesn't seem to exist?